src/com/palmergames/bukkit/towny/TownyMessaging.java

```diff
@@ -50,6 +50,9 @@ public static void sendErrorMsg(Object sender, String msg) {
 		boolean isPlayer = false;
 		if (sender instanceof Player) {
 			isPlayer = true;
+		} else if (sender instanceof Resident) {

```

Comment:
All of the changes to TownyMessaging can be reverted, I don't believe you end up sending anything to TownyMessaging via a resident.

Reply:
In the NationCommand class when you add an enemy, if the list of enemies is empty, you send an error message through TownyMessaging, passing the resident as the sender. It kept popping up as a ClassCastException on my end, since Resident can't be cast to CommandSender

Reply:
Yeah, I see it down there now, not sure why it's working fine on my ide but popping up the ClassCastException for you. Maybe revert your townymessaging and if there's something after I accept it that pops up on my end I'll address it.

Reply:
It only pops up the error in the chat when that list comes up empty. It’s not the IDE throwing the error. But I’ll revert it when I get the chance later tonight.